packages/core/app/initializers/spree-ember-core.js

```diff
@@ -14,11 +13,6 @@ export function initialize(container, application) {
   application.inject('route', 'spree', 'service:spree');
   application.inject('controller', 'spree', 'service:spree');
   application.inject('component', 'spree', 'service:spree');
-
-  /* Copy Environment and Spree Configuration to Spree Service */
-  var SpreeService = container.lookup('service:spree');
-  SpreeService.set('environment', ENV.environment);
-  SpreeService.set('config', ENV['spree'] || {});
 }
```

Comment:
Probably I was trying to extract an instance initializer, but didn't complete the task. I'll put that back and give it another go later.